src/main/resources/META-INF/plugin.xml

```diff
@@ -1,6 +1,6 @@
-<idea-plugin version="2">
+<idea-plugin>
     <id>org.axonframework.intellij.axonplugin</id>
-    <name>Axon Framework Plugin</name>
+    <name>Axon Framework</name>
     <version>${project.version}</version>
     <vendor email="intellijplugin@axonframework.org" url="http://www.axonframework.org/">Axon Framework</vendor>
```

Comment:
I think we should change the URL here. Guessing axoniq.io would be best suited at the moment.

Reply:
Done. I've also removed the email and put the Github issues URL in the README